src/styles/block/stars.scss

```diff
@@ -0,0 +1,36 @@
+.stars {
+  justify-content: space-between;
+  display: flex;
+  margin: 16px 16px 0;
+
+  &__star {
+    display: flex;
+
+    &-each {
+      background-image: url(../images/star.svg);
+      background-position: center;
+      height: 16px;
+      width: 16px;
+      background-repeat: no-repeat;
+      margin-left: 4px;
+    }
+
+    &-each:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
```

Comment:
The selector '&-each:nth-child(-n + 4)' should be replaced with a more appropriate one that reflects the functionality described in the task. This code will always highlight the first four stars, regardless of the rating value. Consider using a class modifier to control the number of active stars based on the rating.